### PR TITLE
fixes #56 and #158

### LINKED ISSUE #56: Clean up Linux tab of Command Line Basics
The Linux tab at the [Command Line Basics](http://tessel.github.io/t2-start/cmd.html) page uses Mac descriptions, but Windows command line screenshots, this should get its proper, linux/unix-based  instructions.

### PR DESCRIPTION
**Pull request by brihijoshi**
Still in progress which fixes #56. Added Linux Terminal images and icon.

### CODE REVIEW

**Comment by mittalshravika** [comment]
The commits made by me(@mittalshravika) are for fixes #158. Added certain links for the reader to connect with the cli documentation.

**Comment by brihijoshi** [comment]
@Frijol PR ready for review.

**Comment by Frijol** [comment]
This PR would also be better as two PR's!

File: blinky.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ In your command line, make a folder for your Tessel code, then initialize a Tess
 
 `t2 init`
 
+Have a look at [the cli docs](https://tessel.gitbooks.io/t2-docs/content/API/CLI.html#starting-projects) to explore `t2 init` command.
```
Comment (Frijol) [inline blinky.md]: One of the most important design factors in Tessel's first run is that getting started doesn't take very long. This means the design and instructions are intentionally very streamlined and uncluttered– making https://github.com/tessel/t2-start/issues/158 a design challenge! We need to make sure that the information is available, but does not feel like an extra step users have to take before moving on.

With this current suggestion, going to look at the docs is a part of the main instructions– but it's not actually necessary that they do it before moving forward. How can we make the distinction clear?

In some places, we use italic text to provide further information:
screen shot 2017-07-24 at 1 53 49 pm

In other places, we call out additional non-necessary exercises with bold bonus annotation:
screen shot 2017-07-24 at 1 54 37 pm

Do either of these approaches make sense for adding in extra docs links? Or, do we need to add a new design element, like a callout box or one of these?
helptext box

I think personally I'd try italic text and see if that works; you are welcome to experiment with other ideas too.

Comment (Frijol) [inline blinky.md]: copy edit, should read: "to explore the `t2 init` command`

File: wifi.md [hunk 2]
```diff
@@ -34,6 +34,8 @@ The password flag is optional. Tessel 2 will save network credentials and attemp
 
 Wifi connectivity is indicated by the on-board amber Wifi LED, which will blink with every packet it receives.
 
+To explore more options available for `t2 wifi`, take a look at [the cli docs](https://tessel.gitbooks.io/t2-docs/content/API/CLI.html#using-wifi).
```
Comment (Frijol) [inline wifi.md]: See note on `t2 init`

File: wifi.md [hunk 3]
```diff
@@ -84,6 +86,8 @@ Great! now that we're connected and authorized, let's check for updates. Run:
 
 This command will check for a new version of the Tessel 2 firmware. If there is an update available, it will be downloaded and installed.
 
+Have a look at [the cli docs](https://tessel.gitbooks.io/t2-docs/content/API/CLI.html#updating-tessel-2-on-board-osfirmware) to get more information on updations.
```
Comment (Frijol) [inline wifi.md]: See note on `t2 init`

**Review by Frijol — CHANGES_REQUESTED** [review]
Fix for #56 is ready to go; fix for #158 needs a bit of work

File: blinky.md (hunk 1 above)
Comment (mittalshravika) [reply blinky.md]: Sure. Will make the changes.

**Comment by mittalshravika** [comment]
Made the changes accordingly. @Frijol you may review the same.

File: blinky.md [hunk 4]
```diff
@@ -18,6 +18,8 @@ In your command line, make a folder for your Tessel code, then initialize a Tess
 
 `t2 init`
 
+_Have a look at [the cli docs](https://tessel.gitbooks.io/t2-docs/content/API/CLI.html#starting-projects) to explore the `t2 init` command._
```
Comment (Frijol) [inline blinky.md]: "CLI" should be capitalized

File: wifi.md [hunk 5]
```diff
@@ -34,6 +34,8 @@ The password flag is optional. Tessel 2 will save network credentials and attemp
 
 Wifi connectivity is indicated by the on-board amber Wifi LED, which will blink with every packet it receives.
 
+_To explore more options available for the `t2 wifi` command, take a look at [the cli docs](https://tessel.gitbooks.io/t2-docs/content/API/CLI.html#using-wifi)._
```
Comment (Frijol) [inline wifi.md]: CLI caps again

File: wifi.md [hunk 6]
```diff
@@ -84,6 +86,8 @@ Great! now that we're connected and authorized, let's check for updates. Run:
 
 This command will check for a new version of the Tessel 2 firmware. If there is an update available, it will be downloaded and installed.
 
+_Have a look at [the cli docs](https://tessel.gitbooks.io/t2-docs/content/API/CLI.html#updating-tessel-2-on-board-osfirmware) to get more information on updations._
```
Comment (Frijol) [inline wifi.md]: CLI caps again

**Review by Frijol — CHANGES_REQUESTED** [review]
Looks good, just a capitalization nitpick

**Comment by mittalshravika** [comment]
Capitalized CLI as suggested.

**Review by Frijol — APPROVED** [review]

**Comment by Frijol** [comment]
Looks great! Merging.

**Pull request merged**